### ISSUE TITLE
delete_series -- influxdb.client.InfluxDBClientError: 400: ... syntax error, unexpected OPERATION_EQUAL, expecting $end

### ISSUE THREAD

**Issue by bmatsuo** [influxdb/influxdb-python#87]
I get the an exception when trying to delete a series using the python client. Here is a shell session demonstrating the problem. Based on the docs it looks like I'm using the module correctly.

```
>>> series = client.get_list_series()
>>> series
[u'grafana.dashboard_Y2FjaGUtbWV0cmljcw==', u'grafana.dashboard_Y2x1c3Rlci1oZWFsdGg=', u'grafana.dashboard_YWJvdXQ=', u'grafana.dashboard_d2VkZ2Vz', u'grafana.dashboard_dGFzay1tZXRyaWNz']
>>> client.delete_series(series[0])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/influxdb/client.py", line 418, in delete_series
    expected_response_code=204
  File "/usr/local/lib/python2.7/dist-packages/influxdb/client.py", line 161, in request
    raise InfluxDBClientError(response.content, response.status_code)
influxdb.client.InfluxDBClientError: 400: Error at 2126212:48 2126212:49. syntax error, unexpected OPERATION_EQUAL, expecting $end
>>> 
```

**Comment by czchen** [comment]
I have a similar problem with UTF-8 series name. According to <http://influxdb.com/docs/v0.8/api/query_language.html#getting-series-with-special-characters>, double quotes is the way to handle special character in series. So I think the solution to double quoted series name is okay as long as double quote escaping is also implemented.

**Comment by ReAzem** [comment]
@czchen You are right. I will work on implementing this properly then.

**Comment by ReAzem** [comment]
Unless someone wants to fix this, I will focus on ```v0.9.0```, where deleting series will be different. However, I will gladly review and merge a PR that fixes it for the current version of influxdb.
